data/en/test_multiple_piping.json

```diff
@@ -117,14 +107,14 @@
                     "id": "piping-question",
                     "questions": [{
                         "id": "multiple-piping-question",
-                        "title": "Does <em>{{[answers.first_text, answers.second_text] | format_household_name}}</em> live at <em>{{answers.address_line_1}}</em>",
+                        "title": "Does <em>{{[answers['first-text'], answers['second-text']] | format_household_name}}</em> live at <em>{{answers['address-line-1']}}</em>",
                         "type": "General",
                         "answers": [{
                             "id": "multiple-piping-answer",
                             "type": "Radio",
                             "mandatory": false,
                             "options": [{
-                                    "label": "{{[answers.address_building, answers.address_line_1, answers.address_line_2, answers.address_line_3, answers.address_town_city, answers.address_county, answers.address_postcode, answers.address_country] | concatenated_list}}",
+                                    "label": "{{[answers.building, answers['address-line-1'], answers['address-line-2'], answers['address-line-3'], answers['town-city'], answers['county'], answers['postcode'], answers['country']] | concatenated_list}}",
```

Comment:
This PR is only changing ones where the answer id is hyphenated but do we want these two standards or try and follow the same format of answers['answerID'] regardless of hyphenation? If we had a single rule it would make it easier if we ever want to search a schema for the use of answers in piping as we'd only ever have to search the one format.

Reply:
Hmm, possibly. Although I believe there's already another card being worked on by @MebinAbraham that will take both formats into account.

Reply:
@nealedj @dwyeradam

Reply:
Personally I like to chose a style and stick to it. We know `answers['answer-id']` will work for anything and so that should be our standard. As I said, not a blocker by any measure but simplifies if we say we define answer piping as this. Rather than we define answer piping as this except in this situation where it could also be this but that work work for this.

Reply:
I agree - let's standardise. I'll updated.